src/systems/personal-space-bubble.js

```diff
@@ -177,15 +184,78 @@ AFRAME.registerComponent("personal-space-invader", {
   }
 });
 
+// TODO come up with better visuals than a wireframe sphere
+function createBubbleObject(radius) {
+  const material = new THREE.LineBasicMaterial({ color: 0x2f80ed });
+  const geometry = new THREE.WireframeGeometry(new THREE.SphereBufferGeometry(radius, 20, 20));
+  const sphere = new THREE.LineSegments(geometry, material);
+  material.transparent = true;
+  return sphere;
+}
+
 AFRAME.registerComponent("personal-space-bubble", {
   schema: {
     radius: { type: "number", default: 0.8 },
+    invadeIndicatorCooldown: { type: "number", default: 5000 },
     debug: { default: false }
   },
   init() {
+    this.bubble = createBubbleObject(this.data.radius);
+    this.el.setObject3D("bubble", this.bubble);
+
+    // Only shown when state is toggled or personal space is invaded
+    this.setBubbleVisibility(false);
+
+    this.bubbleOffTween = new TWEEN.Tween(this.bubble.scale)
+      .to({ x: 0.001, y: 0.001, z: 0.001 }, 500)
+      .easing(TWEEN.Easing.Quadratic.In)
+      .onComplete(() => (this.bubble.visible = false));
+
+    this.fadeOutTween = new TWEEN.Tween(this.bubble.material)
+      .to({ opacity: 0 }, 250)
+      .delay(300)
+      .onComplete(() => (this.bubble.visible = false));
+
+    this.bubbleOnTween = new TWEEN.Tween(this.bubble.scale)
+      .to({ x: 1, y: 1, z: 1 }, 600)
+      .easing(TWEEN.Easing.Elastic.InOut)
```

Comment:
You might like Quadratic.Out so that it quickly gets out of your face and then slowly settles into the scale it ends up at.